packages/fast-animation/lib/animate/index.ts

```diff
@@ -156,7 +156,7 @@ export default abstract class Animate {
     public set onFinish(callback: () => void) {
         this._onFinish = callback;
 
-        if (Boolean(this.animation)) {
+        if (this.animation) {
```

Comment:
I'm curious as to why this was an issue.

Reply:
Should be okay